src/include/nmtkit/formatter.h

```diff
@@ -40,6 +40,9 @@ class Formatter {
   //   os: Target output stream.
   virtual void finalize(std::ostream * os) = 0;
 
+  // TODO write explanation
+  virtual void setReferencePath(std::string ref_file_path) = 0;
```

Comment:
This looks a HTMLFormatter-specific option and should not be a general function of Formatter for now.